pkg/structures/models.go

```diff
@@ -0,0 +1,31 @@
+package structures
+
+import (
+	"time"
+)
+
+type ExternalID string
+
+func (eid ExternalID) IsEqual(id ExternalID) bool {
+	return eid == id
+}
+
+type Structure struct {
+	ID          string       `json:"structureID" bson:"structureID"`
+	CreatedAt   *time.Time   `json:"createdAt,omitempty" bson:"createdAt"`
+	ExternalIDs []ExternalID `json:"externalIDs,omitempty" bson:"externalIDs,omitempty"`
+	Site        *Site        `json:"site,omitempty" bson:"site,omitempty"`
+	Type        string       `json:"type,omitempty" bson:"type,omitempty"`
+	UpdatedAt   *time.Time   `json:"updatedAt,omitempty" bson:"updatedAt"`
+}
+
+type Site struct {
```

Comment:
Is it the site identifier that is required for something?

Reply:
i...uhh...am not sure i understand your question...